tests/toolkit/test_utils.py

```diff
@@ -0,0 +1,56 @@
+from goose.toolkit.utils import parse_plan
+
+
+def test_parse_plan_simple():
+    plan_str = (
```

Comment:
just curious: is this our recommended style for multiline strings? what are the trade offs for this over triple-quoted multiline strings or even keeping the markdown as a fixture?

Reply:
to be honest, I did it to pass the ruff checks. Not sure if we've discussed it before